ci/jobs/macros.yaml

```diff
@@ -5,6 +5,15 @@
 # they're easy to find.
 ###
 
+# Inject the PYTHONUNBUFFERED env var into builds using a parameter
+- parameter:
+    name: python-unbuffered
+    parameters:
+      - string:
```

Comment:
Would not be better to use a boolean? It will end up with true or false as a string on the variable.